app/src/main/res/layout/save_backup_layout.xml

```diff
@@ -5,122 +5,112 @@
     android:id="@+id/backup_dialog"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@drawable/background_card"
-    android:elevation="10dp"
-    android:theme="@style/MaterialAppTheme"
-    android:visibility="invisible"
-    tools:visibility="visible"
+    android:background="@drawable/rounded_dialog"
+    android:clipChildren="false"
+    android:clipToPadding="false"
+    android:orientation="vertical"
+    android:paddingStart="25dp"
+    android:paddingEnd="0dp"
     >
-
   <TextView
       android:id="@+id/dialog_title"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginStart="14dp"
-      android:layout_marginTop="20dp"
+      android:layout_marginTop="@dimen/big_margin"
       android:fontFamily="sans-serif-medium"
       android:letterSpacing="0.01"
       android:lineSpacingExtra="6sp"
       android:text="@string/backup_save_dialogue_title"
-      android:textColor="@color/black"
+      android:textColor="#011728"
```

Comment:
we shouldn't have hardcoded colors in the layout, even if it is only being used here.. to make it easier to reuse the color

Reply:
agreed, altered